package.json

```diff
@@ -305,6 +306,7 @@
     "concurrently": "^5.2.0",
     "css-element-queries": "^1.2.3",
     "css-loader": "^3.5.3",
+    "deepdash": "^5.3.5",
```

Comment:
This is used to recursive search within a complex object. Used inside `<TreeView/>` to look for selected item.